test/e2e/rbac_test.go

```
@@ -0,0 +1,343 @@
+	}
+	if err != nil {
+		t.Fatalf("Failed to create framework: %v", err)
+	}
+
+	ctx := context.Background()
+
+	nsY, cleanupY, err := f.CreateNamespace(ctx, "test-rbac-create-y", false)
+	if err != nil {
+		t.Fatalf("Failed to create namespace Y: %v", err)
+	}
+	defer cleanupY()
```

golangci-lint (2.12.2)

[error] Error return value is not checked

(errcheck)